docs/guides/vm-multi-region-setup/README.md

```diff
@@ -0,0 +1,202 @@
+
+The setup process will be outlined in three stages:
+- Set up a self-contained cluster in a single AZ.
+- Expand the setup to multi AZ.
+- Set up vmagent and vmauth.
+
+## Self-contained cluster in single AZ
+
+A minimal VictoriaMetrics cluster in the AZ must contain the following nodes:
+- a single `vmstorage` node
+- a single `vminsert` node with `-storageNode=<vmstorage-host>`
+- a single `vmselect` node with `-storageNode=<vmstorage-host>`
+
+For high availability purposes, it is recommended to run at least two nodes for each service.
+
+### vmstorage
+Execute the following command to start vmstorage process. (please refer to [List of command-line flags for vmstorage](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#list-of-command-line-flags-for-vmstorage) for more command-line flags)
+```sh
+/path/to/vmstorage-prod
+```
+You can see those in the output:
+```text
+lib/vminsertapi/server.go:60        started TCP vminsert server at "0.0.0.0:8400"
+lib/vminsertapi/server.go:83        accepting vminsert conns at 0.0.0.0:8400
+lib/vmselectapi/server.go:156        accepting vmselect conns at 0.0.0.0:8401
+lib/httpserver/httpserver.go:145        started server at http://0.0.0.0:8482/
+```
+The default ports `:8400` and `:8401` are designated for data ingestion (from `vminsert`) and query (from `vmselect`), respectively.
+You can start several `vmstorage` processes on different hosts to obtain `<az1-vmstorage-host[1-N]>`.
+
+### vmselect
+Execute the following command to start `vmselect` process. (please refer to [List of command-line flags for vmselect](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#list-of-command-line-flags-for-vmselect) for more command-line flags).
+
+```sh
+/path/to/vmselect-prod -storageNode=<az1-vmstorage-host1>:8401
+```
+If you have started several `vmstorage` instances before, you can separate them with commas in `-storageNode`:
+```sh
+/path/to/vmselect-prod \
+    -storageNode=<az1-vmstorage-host1>:8401,<az1-vmstorage-host2>:8401
+```
+You can start several `vmselect` processes on different hosts to obtain `<az1-vmselect-host[1-N]>`.
+
+### vminsert
+
+Execute the following command to start `vminsert` process. (please refer to [List of command-line flags for vminsert](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/#list-of-command-line-flags-for-vminsert) for more command-line flags)
+```sh
+/path/to/vminsert-prod -storageNode=<az1-vmstorage-host1>:8400
+```
+In the multi-AZ cluster topology, we don’t need to set `-replicationFactor` for `vminsert`,  data shoulD be replicated between these AZ clusters.
+
+If you have started several `vmstorage` instances, you can separate them with commas in `-storageNode`:
+```sh
+/path/to/vminsert-prod \
+    -storageNode=<az1-vmstorage-host1>:8400,<az1-vmstorage-host2>:8400
+```
+You can start several such `vminset` processes on different hosts to obtain `<az1-vminsert-host[1-N]>`.
```

Comment:
I don't think we need to include elaborate commands for vmcluster components in this document, as the focus here is on a higher level. It would be preferable to focus on communication across multiple zones and refer the setup of a single vmcluster to other documentation if needed.